aya-obj/README.md

```diff
@@ -0,0 +1,55 @@
+# aya-obj
+
+## Status
+
+NOTE: this crate includes code that started as internal API used by
```

Comment:
I think you can remove NOTE: and just start with This crate...

Reply:
Fixed. Thanks!